WeatherNotFound/src/main/java/me/learning/weathernotfound/presentation/WeatherNotFound.kt

```diff
@@ -66,7 +68,6 @@ class WeatherNotFound private constructor() {
      */
     fun init(
         context: Context,
-        openWeatherApiKey: String,
```

Comment:
Imagine developer doesn't want to initialize library in manifest at all, in this scenario, they expect to fully initialize library within java or kotlin code. at the end I think it's good to receive `openWeatherApiKey` from this function.

please revert the changes or comment your idea about this.